.github/workflows/ci.yml

```diff
@@ -116,31 +112,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Install Uncrustify
-        run: sudo apt-get install uncrustify
-      - name: Run Uncrustify
-        run: find . -iname "*.[hc]" -exec uncrustify --check -c tools/uncrustify.cfg {} +
-      - name: Check For Trailing Whitespace
-        run: |
-          set +e
-          grep --exclude="README.md" -rnI -e "[[:blank:]]$" .
-          if [ "$?" = "0" ]; then
-            echo "Files have trailing whitespace."
-            exit 1
-          else
-            exit 0
-          fi
-            url-check:
-  url-check:
+      - name: Check formatting
+        uses: FreeRTOS/CI-CD-Github-Actions/formatting@main
+        with:
+          path: ./
+          exclude-dirs: .git
+  link-verifier:
     runs-on: ubuntu-latest
     steps:
-      - name: Clone This Repo
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+      - name: Setup Python for link verifier action
+        uses: actions/setup-python@v2
         with:
-            path: ./kernel
-      - name: URL Checker
-        run: |
-            bash kernel/.github/actions/url_verifier.sh kernel
+          python-version: '3.8'
+      - name: Check Links
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does the URL checker need a GITHUB_TOKEN?

Reply:
The link verifier script uses the `gh` client for checking Issues and PRs. We don't necessarily need it. https://github.com/FreeRTOS/CI-CD-Github-Actions/tree/main/link-verifier#pre-requisites

Reply:
Nevermind. I understand the reason this is required..